doc/OnlineDocs/modeling_extensions/gdp/solving.rst

```diff
@@ -79,8 +88,9 @@ The Hull Reformulation requires a lifting into a higher-dimensional space and co
 
     - All variables that appear in disjuncts need upper and lower bounds.
 
-    - The hull reformulation is an exact reformulation at the solution points
-      even for nonconvex GDP models, but the resulting MINLP will also be nonconvex.
+    - The hull reformulation is an exact reformulation at the solution
+      points even for nonconvex GDP models, but the resulting MINLP will
```

Comment:
This is debatable. I would not go into this rabbit hole. I would write:
The Hull reformulation is based on a high-dimensional representation of the disjunctive sets, which yields the convex hull of the disjunctions when they represent convex sets.